wandb/sdk/launch/agent/agent.py

```diff
@@ -179,5 +199,10 @@ def loop(self) -> None:
                     continue
                 self.run_job(job)
         except KeyboardInterrupt:
+            shutdown_update = self._api.update_launch_agent_status(
+                self._id, AGENT_KILLED
```

Comment:
Just a note, we'll need to make a recurring jobs to mark agents as killed in the back end.

Reply:
wait why doesn't this work? or are you talking about the "edge" cases where an agent hangs or dies for some other reason, then yeah def

Reply:
Yeah, all I'm saying is we probably want to clean up killed agents to keep the table from growing indefinitely